packages/extension/src/newtab/App.tsx

```diff
@@ -67,7 +74,6 @@ Modal.defaultStyles = {};
 const getRedirectUri = () => browser.runtime.getURL('index.html');
 function InternalApp(): ReactElement {
   useError();
-  useLazyModal();
```

Comment:
We should not need this.

Reply:
Why not? 
Pretty sure it's used to render the modals everywhere right?

Reply:
This is actually a duplicate. There is one more a bit below.

Reply:
Hmm, I'd be very careful probably some super weird edge-case.
Remember it being removed and breaking something.

I see I added the bottom one when I did the firefox privacy addition.
Can't fully recall but I do think it was some weird edge-case of it not calling in time.

If you verified it, it's fine we can try

Reply:
Yeah, this is the same thing I worked on during the re-render optimization week.